sky/packages/sky/lib/src/fn3/framework.dart

```diff
@@ -599,13 +601,14 @@ abstract class Element<T extends Widget> implements BuildContext {
   }
 }
 
-typedef Widget WidgetBuilder(BuildContext context);
-typedef void BuildScheduler(BuildableElement element);
-
 class ErrorWidget extends LeafRenderObjectWidget {
   RenderBox createRenderObject() => new RenderErrorBox();
 }
 
+typedef Widget WidgetBuilder(BuildContext context);
+typedef void BuildScheduler(BuildableElement element);
+typedef void FunctionBody();
```

Comment:
I'd remove this declaration and just declare it inline the one place it is used.